### DISCUSSION TITLE
[v4] Setting CSS variable shorthand

### DISCUSSION THREAD

**Discussion by jacksleight** [tailwindlabs/tailwindcss#15379] [Help] [answered]
In v3 you could do this:

```
[--foo:--bar]
```

Which would result in:

```
--foo: var(--bar);
```

In v4 it just gives:

```
--foo: --bar;
```

I know the way you have to use CSS variables [has changed](https://tailwindcss.com/docs/v4-beta#css-variables-in-arbitrary-values), but I can't figure out how to use that new syntax with an arbitrary property. Is it still possible?

**Comment by wongjn** [comment]
Try `[--foo:var(--bar)]`.

**Reply by jacksleight** [reply]
Yeah, I'm doing that for now. Was just wondering if the even shorter syntax was still supported.

**Reply by adamwathan** [reply, marked as answer]
Right now no shorter way I'm afraid!
